example/pubspec.yaml

```diff
@@ -22,14 +22,16 @@ dependencies:
 #  fhir_auth:
 #    path: ../../fhir/fhir_auth
 
+  file: ^6.1.4
+
   flutter:
     sdk: flutter
   flutter_localizations:
     sdk: flutter
 
-  google_fonts: ^2.1.1
```

Comment:
This change is necessary since after upgrading intl, for some reason, the project can't be built.